### DISCUSSION TITLE
Security Onion Not Updating Properly?

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#5319] [Unsupported Versions]
Hello,

After transferring 2.3.70 iso from workstation to airgaped manager I ran soup and pointed it towards the correct directory where the iso is located. 

After running soup the version still remains 2.3.61. This is an airgaped SOv2 deployment. Any suggestions as to why the grid is not updating to 2.3.70 After soup?

![20210827_100756](https://user-images.githubusercontent.com/70167373/131148028-e4f4b13a-8caa-4dc4-9da9-0537089ad1de.jpg)

**Comment by dougburks** [comment]
Is it possible that you still have an old ISO mounted?

**Reply by ColeVan** [reply]
Hey Doug,

I suspected the same thing. In the screen shot you can see that 2.3.70 iso is located in the folder. I pointed soup to the correct folder location with updated iso and recieved the same outcome saying 2.3.61 is up to date. Any suggestions?

**Reply by dougburks** [reply]
Looking at the screenshot, I don't see `soup` asking for the path to the ISO.

How exactly did you point `soup` to the ISO?

Have you tried running the `mount` command to see what is actually mounted?
